lib/HTTPStore/index.js

```diff
@@ -200,16 +210,40 @@ class HTTPStore extends Store {
    * @private
    * @param {String} path The path that will be used to match the observer.
    * @param {Object} observer The observer to remove.
-   * @return {HTTPStore} The instance of the current HTTPStore.
+   * @return {HTTPStore} Instance of the current HTTPStore.
    */
   _unobserve(path, observer) {
-    const [, refreshHandle] = observer;
-    if(refreshHandle !== null) {
-      clearInterval(refreshHandle);
-    }
     this.data.get(path).observers.delete(observer);
     return this;
   }
+
+  /**
+   * Returns an options object build with the global store options and the path specific options if path is defined.
+   * @param  {String} [path] Path of the path specific options.
+   * @return {Object} Options object.
+   * @param {Object} query Query that will be use to fetch the store and set the state.
+   */
+  _getOptions(path) {
+    return Object.assign({}, this._options, path ? this._findOrCreatePathData(path).options : {});
+  }
+
+  /**
+   * Updates options
+   * @param  {Object|null} [query] Query of the query specific options.
+   * @param  {Object} options Options to merge with the current options.
+   * @return {HTTPStore} Instance of the current HTTPStore.
+   */
+  options(query, options) {
```

Comment:
I am not completely sure that allowing to set options per query is a good idea after all. It adds a little overhead with `_getOptions` and I can't remember a legit usage. What do you think @elierotenberg ?